### ISSUE TITLE
Development on M1 chips - do we still need Docker?

### ISSUE THREAD

**Issue by newswim** [open-austin/azure-indigent-defense#56]
Some time has passed since we first had to deal with the Docker-based workaround for developing on M1-based MacBooks, I'm wondering if support has changed. Since I don't have an M1, I'm unable to test locally. But I can do some initial investigation and see if MS has increased their support.

**Comment by DevTrav** [comment]
Still seems to be the case @newswim. Here is the official MS support [answer as of June 14 ](https://learn.microsoft.com/en-us/answers/questions/1304451/how-to-develop-python-azure-function-my-mac-with-c)
I also confirmed that ARM64 remains unsupported within the [current Azure docs](https://learn.microsoft.com/en-us/azure/azure-functions/functions-run-local?tabs=macos%2Cportal%2Cv2%2Cbash&pivots=programming-language-csharp#x86-emulation-on-arm64)

Tl;dr
`Functions doesn't currently support local Python function development on ARM64 devices....develop Python functions on a Mac with an M1 chip by running in an emulated x86 environment.`

It was my experience that Docker was the happiest path for emulation fwiw

**Comment by nareddyt** [comment]
TBH I'd recommend using the docker installation for any developer. Docker is nice because it programmatically pulls the entire toolchain needed to run our functions. Whereas local dev without docker requires users to install all CLIs and dependencies manually.

For these reasons, in https://github.com/open-austin/azure-indigent-defense/pull/55, I removed the old local instructions and recommended everyone uses docker for consistency.

**Comment by newswim** [comment]
I totally agree, I think we can close this one.
